app/uploaders/achievement_badge_uploader.rb

```diff
@@ -0,0 +1,13 @@
+# Image uploader for Achievement Badge
+# This class contains achievement badge specific settings for the image uploader.
+class AchievementBadgeUploader < ImageUploader
```

Comment:
For resizing, I think the three versions thumb (50 * 50), small (100* 100) and medium (200 * 200) are general enough to suite most of the cases. And we will use css to resize the image anyway ?

But if there are other configurations difference (e.g. file stores on different path, using different storage), then a new type of image uploader is needed.

So in most of cases, `ImageUploder` alone is enough IMO.